README.md

```diff
@@ -178,8 +179,9 @@ You can deploy this project using:
 
 > Describe 1 - 3 features you will add to the project.
 
```

Comment:
- [ ] Please notice the section in your readme which start with `>` are directions/guidelines from microverse to help you understand what to write in that section and should not be part of your readme file. Please consider removing this and all the like contents from the readme file.